src/views/admin/organisationen/KlassenManagementView.vue

```diff
@@ -324,17 +330,34 @@
               <template v-slot:activator="{ props }">
                 <div v-bind="props">
                   <KlassenFilter
+                    :filterId="'klassen-management-klassen-filter'"
```

Comment:
```suggestion
                    :filterIdPrefix="'klassen-management'"
```
The id can get too long otherwise